File: Python3/Secuenciales/ej12.py
# -*- coding: utf-8 -*-
import math
#Pide al usuario dos pares de números x1,y2 y x2,y2, que representen 
#dos puntos en el plano. Calcula y muestra la distancia entre ellos.

#1.pedir el valor x del punto1
#2.almacenar x1
#3.pedir el valor y del punto1
#4.almacenar y1
#5.pedir el valor x del punto2
#6.almacenar x2
#7.pedir el valor y del punto2
#8.almacenar y2
#9.aplicar fórmula distancia = math.sqrt((x2-x1)**2+(y2-y1)**2)
#10.mostrar distancia

#pedir datos
print("==::Calcular la distancia entre dos puntos cartesianos::==")
print("--Valores del punto 1")
x1 = float(input("Introduce la coordena X del punto1> "))
y1 = float(input("Introduce la coordena Y del punto1> "))
print("--Valores del punto 2")
x2 = float(input("Introduce la coordena X del punto2> "))
y2 = float(input("Introduce la coordena Y del punto2> "))

#calculo
distancia = round(math.sqrt((x2-x1)**2+(y2-y1)**2),2)

#mostrar
print("la distancia entre los puntos "+str(x1)+", "+str(y1)+" y "+str(x2)+", "+str(y2)+" es "+str(distancia))

File: Java/Secuenciales/ej12.java
import java.util.Scanner;

/**
* Pide al usuario dos pares de números x1,y2 y x2,y2, que representen dos puntos en el plano. 
* Calcula y muestra la distancia entre ellos.
* @author Alberto Miguel Martinez Jimenez 
*/

//Almacenar el valor x del punto1
//Almacenar el valor y del punto1
//Almacenar el valor x del punto2
//Almacenar el valor y del punto2
//Aplicar fórmula distancia = Math.sqrt((x2-x1)**2+(y2-y1)**2)
//mostrar distancia

public class ej12 {
	public static void main(String[] args) {
		int x1;
		int y1;
		int x2;
		int y2;
		double distancia;
		Scanner scanner = new Scanner(System.in);//Intanciar el objeto Scanner
		
		System.out.println("Introduce la coordenada X del punto1>");
		x1 = scanner.nextInt();
		
		System.out.println("Introduce la coordenada Y del punto1>");
		y1 = scanner.nextInt();
		
		System.out.println("Introduce la coordenada X del punto2>");
		x2 = scanner.nextInt();
		
		System.out.println("Introduce la coordenada Y del punto2>");
		y2 = scanner.nextInt();
		
		distancia = Math.sqrt(Math.pow((x2-x1),2)+Math.pow(y2-y1,2));
		System.out.println("La distancia entre ambos puntos es "+distancia);
	}
}